questions.md

```diff
@@ -0,0 +1,9 @@
+#Questions:
+
+1. If i want to make a pull request to repo of other user, do i need to clone it first?
+2. In the classroom every studen has a repo, they need to make a pull to download all and create a new branch and stat to work on it and then just push ?
```

Comment:
Correct! If they are using a terminal, it would look like this:

```
git clone [repo-url]
cd [repo-name]
git checkout -b [working-branch-name]
git add MyCode.java
git commit -m "Start MyCode"
git push
```